docker-compose.yml

```diff
@@ -44,8 +44,10 @@ services:
     volumes:
       - ./build/mongod_secured.conf:/etc/mongod.conf
 
-  ferretdb-postgresql:
-    image: ${FERRETDB_IMAGE:-ferretdb-local}
+  ferretdb:
+    build:
+      context: ./build/deps
+      dockerfile: ferretdb.Dockerfile
```

Comment:
Please check what docker images we should test against, `latest` is used for now.

Reply:
We use Dockerfiles instead of just specifying the image name there to make Dependabot update them. So let's use full image tags in Dockerfiles.

Reply:
Added full image name :+1: